src/main/java/net/dries007/tfc/client/screen/PetCommandScreen.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed under the EUPL, Version 1.2.
+ * You may obtain a copy of the Licence at:
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ */
+
+package net.dries007.tfc.client.screen;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.components.Button;
+import net.minecraft.client.gui.components.Widget;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.world.item.ItemStack;
+
+import net.minecraftforge.network.PacketDistributor;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import net.dries007.tfc.client.RenderHelpers;
+import net.dries007.tfc.common.entities.livestock.pet.TamableMammal;
+import net.dries007.tfc.network.PacketHandler;
+import net.dries007.tfc.network.PetCommandPacket;
+import net.dries007.tfc.network.ScreenButtonPacket;
+import net.dries007.tfc.util.Helpers;
+
+public class PetCommandScreen extends Screen
+{
+    private final TamableMammal entity;
+
+    public PetCommandScreen(TamableMammal entity)
+    {
+        super(Helpers.translatable("tfc.screen.pet_command"));
+        this.entity = entity;
+    }
+
+    @Override
+    protected void init()
+    {
+        super.init();
+        int y = 72;
+        for (TamableMammal.Command command : TamableMammal.Command.VALUES)
+        {
+            if (entity.willListenTo(command, true))
+            {
+                addRenderableWidget(new Button(width / 2 - 100, height / 4 + y, 200, 20, Helpers.translateEnum(command), b -> {
+                    PacketHandler.send(PacketDistributor.SERVER.noArg(), new PetCommandPacket(entity, command));
+                    Minecraft.getInstance().setScreen(null);
```

Comment:
I believe you also need a `playerInventory.player.containerMenu = playerInventory.player.inventoryMenu;` here. It probably doesn't matter, as I don't think this container has slots, but it still "closes" the container on client.